src/components/Portal/Portal.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+import { prop } from "styled-tools";
+import as from "../../enhancers/as";
+import Base from "../Base";
+
+class Component extends React.Component {
+  state = {};
```

Comment:
Is this a replacement for calling the constructor?

Reply:
That's it.